frontend/ui/screensaver.lua

```diff
@@ -410,8 +431,28 @@ function Screensaver:show(event, fallback_message)
                 }
             }
         end
+
         -- No overlay needed as we just displayed the message
         overlay_message = nil
+
+        -- check if message_widget should be overlaid on another widget
+        if message_widget ~= nil then
```

Comment:
The reverse works well enough.
```suggestion
        if message_widget then
```